lib/transport/memory.js

```diff
@@ -33,12 +33,19 @@ memory.request = function(action, payload, done) {
     });
 
     function complete(e, results) {
-      var o = {
-      };
-      _.each(results, function(r) {
-        o = _.merge(o, r);
+      var result = _.reduce(results, function(accum, item) {
+        if (_.isEmpty(item)) {
+          return accum;
+        }
+
+        if (_.isEmpty(accum)) {
+          return item;
+        }
+
+        return _.merge(accum, item);
```

Comment:
This should be `_.assign` if you just want a shallow merge, will be faster. @peterjwest